assets/js/workflow-reducer.js

```diff
@@ -183,17 +217,74 @@ export function setWorkflowPublicAction(workflowId, isPublic) {
   }
 }
 
+// REORDER_WFMODULES
+// Re-order the modules in the module stack
+export function reorderWfModulesAction(wfModuleID, newIndex) {
```

Comment:
Is this still used? I thought you only reorder when the drop ends.
If you keep it, it needs tests.